tests/chainer_tests/training_tests/updaters_tests/test_standard_updater.py

```diff
@@ -70,55 +70,53 @@ def setUp(self):
             self.iterator, self.optimizer)
 
     def test_init_values(self):
-        self.assertIsNone(self.updater.device)
-        self.assertIsNone(self.updater.loss_func)
-        self.assertEqual(self.updater.iteration, 0)
+        assert self.updater.device is None
+        assert self.updater.loss_func is None
+        assert self.updater.iteration == 0
 
     def test_epoch(self):
-        self.assertEqual(self.updater.epoch, 1)
+        assert self.updater.epoch == 1
 
     def test_new_epoch(self):
-        self.assertTrue(self.updater.is_new_epoch)
+        assert self.updater.is_new_epoch is True
```

Comment:
```suggestion
        assert self.updater.is_new_epoch
```

Reply:
I think `is True` is better for testing.
`assert self.updater.is_new_epoch` would succeed if it is `5` or a function or anything else.

Reply:
OK, I understand.